### ISSUE TITLE
Unable to login with PIN (Windows Hello)

### ISSUE THREAD

**Issue by hwalker928** [ChrisTitusTech/winutil#886]
**Describe the bug**
After running the Laptop tweaks, I was unable to login to my laptop with my Windows Hello PIN, with the status code of 0xc000006d. Windows requested me to change my PIN, of which fixed the issue.

**To Reproduce**
1. Run the default Laptop tweaks using winutil
2. Restart the device
3. Attempt to login using a PIN

**Expected behavior**
Should be able to login. 

**Screenshots**
No screenshots currently available as I resolved the issue before realising a potential issue with the tweaks. 

**Additional context**
I did also notice that the Netlogon service was set to Manual as a result of running the tweaks.  I have only noticed the issue on one device, as that is all I have used the script on, however this may be completely coincidental regarding when I ran the tweaks. Additionally, I had ran Winutil a few months prior to now, of which logon worked fine.

**Comment by ChrisTitusTech** [comment]
I'm unable to replicate this as I use windows hello on my system with tweaks and it still works. 
I'd recommend resetting it using the settings troubleshooter.

**Comment by alexbalandi** [comment]
Sorry for writing to closed issue, but I ran into the same problem after running the tweak and what's worse : I can't reenable the pin.
Here's what happens if i try to set up in powershell:
```
PS C:\Windows\system32> net user alexb *
System error 8646 has occurred.                                                                                                                                                                                                                 The system is not authoritative for the specified account and therefore cannot complete the operation. Please retry the operation using the provider associated with this account. If this is an online provider please use the provider's online site.              
```
And I also tried every single advice from here with no luck (ui says "something went wrong" when I try to set up a pin)  : https://www.partitionwizard.com/clone-disk/cant-add-pin-windows-10.html

I know it's on me to not make the backup and properly remember which tweaks I used (mostly telemetry and turning services to manual run), but I would really appreciate if you can give me any advice on where to look to resolve the issue.

If I were to guess, I'd say something that should be setting up the PIN doesn't have enough rights, but I seemingly made the relevant directory very persmissive and fixed its ownership, so I'm out of guesses where the root of the problem lies.

**Comment by dmitriy1980** [comment]
Look into #2 under https://www.thewindowsclub.com/cannot-change-or-modify-pin-in-windows. It helped me 2 weeks ago.

**Comment by alexbalandi** [comment]
> Look into #2 under https://www.thewindowsclub.com/cannot-change-or-modify-pin-in-windows. It helped me 2 weeks ago.

Big thanks for chiming in, but nah - sadly, this didnn't work :( Strange thing is I made a new user on my PC and I can set up PIN for it just fine. But my original account can't do the same thing :/ I will keep digging I guess.

**Comment by alexbalandi** [comment]
I think I pinpointed an issue a bit more. I can set up PIN fine with local account (or if I switch to local), but theres no way to do it for me if I use my microsoft account and sign in with it.

@ChrisTitusTech when testing this, did you use local account?

**Comment by alexbalandi** [comment]
After loading in safe mode (to fix shooting myself in the foot after first using `net user administrator /active:yes` and then `net user administrator /active:no` leaving me with no admin rights) and changing in properties the group of my main account back to admin - I finally could set up PIN in my normal load.
So probably using the ubilt-in admin account to manually set the group of my main account somehow did the trick?
I have no idea, but will leave this here.

**Comment by ChrisTitusTech** [comment]
> 



> I think I pinpointed an issue a bit more. I can set up PIN fine with local account (or if I switch to local), but theres no way to do it for me if I use my microsoft account and sign in with it.
> 
> @ChrisTitusTech when testing this, did you use local account?

I did use a local account on initial setup, but my microsoft account is also setup. I'll keep an eye on this to see if I see more issue pop up.

**Comment by achraf-sani** [comment]
I also ran into this issue today when running the laptop tweaks, I can't login using Windows Hello PIN and trying to reset it didn't fix it. I had a Microsoft account initially setup.

**Comment by ChrisTitusTech** [comment]
Thank you @UnlucksMcGee for the report I fixed this in commit d27345c
